cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -90,6 +91,7 @@ func NewCmdApply(apf *applyPlanFlags) *cobra.Command {
 	cmd.Flags().BoolVarP(&flags.force, "force", "f", flags.force, "Force upgrading although some requirements might not be met. This also implies non-interactive mode.")
 	cmd.Flags().BoolVar(&flags.dryRun, options.DryRun, flags.dryRun, "Do not change any state, just output what actions would be performed.")
 	cmd.Flags().BoolVar(&flags.etcdUpgrade, "etcd-upgrade", flags.etcdUpgrade, "Perform the upgrade of etcd.")
+	cmd.Flags().BoolVar(&flags.renewCerts, "certificate-renewal", flags.etcdUpgrade, "Perform the renewal of certificates used by component changed during upgrades.")
```

Comment:
`flags.etcdUpgrade` -> `true`?
i also think we should default the value to `true`.
[1]

Reply:
@neolit123 great catch!
Now it should be fixed